seed.yaml.sample

```diff
@@ -0,0 +1,10 @@
+Keys:
+  Asymmetric:
+    Rsa:
+      - Metadata:
+          KeyId: "REPLACE_ME_KEY_ID_HERE"
```

Comment:
The key ID will need to be the same as this value: https://github.com/alphagov/di-authentication-account-management/blob/main/src/config/aws.ts#L10

Reply:
Ok cool, I'll hard code that in then

Reply:
Or... actually humm...

Having this hard coded in two places kind of sucks.
It might be better if [this line](https://github.com/alphagov/di-authentication-account-management/blob/main/src/config/aws.ts#L10) loaded the YAML and took the KeyID from this file, no?

Reply:
point about not hard coding it in seeds.yaml is that someone could register a new client_id with auth build?

Reply:
Setting the key ID is just to make sure the app is using the correct key when it calls out to localstack - that ID won't need to change if someone registers a new client to use locally. 

I see what you mean about having to set it in two places but we shouldn't ever have to change this unless we completely change how we run this app locally. I think it's fine as is in this case

Reply:
Ok gotcha, that's a lot easier then

Reply:
Done
https://github.com/alphagov/di-authentication-account-management/pull/683/files#diff-bbbe0411a120b1a494f185ef61d6256dddd80c9176ae4485a22f62a4de76bc00R5